common/keys.go

```diff
@@ -205,3 +205,4 @@ const (
 	// ArgoCDArgoprojKeyManagedByClusterArgoCD is needed to identify namespace mentioned as sourceNamespace on ArgoCD
 	ArgoCDArgoprojKeyManagedByClusterArgoCD = "argocd.argoproj.io/managed-by-cluster-argocd"
 )
+
```

Comment:
no change in this file except EOF. Can we revert this file just to simplify the history?

Reply:
Reverted.